perl/README.md

```diff
@@ -0,0 +1,64 @@
+# Bystro Perl package
+
+## Installing Bystro using docker
+
+To build a docker image using the `Dockerfile`, run the following:
+
+```bash
+docker build --tag bystro-cli .
+```
+
+## Installing Bystro dependencies locally
+
+Assuming that you've cloned the repository and are working on it locally, then the dependencies can mostly be installed with cpanm.
+But there are a few one-off dependencies that require a slightly modified approach.
+
+One-off dependencies can be installed as follows:
+
+```bash
+cpanm --quiet https://github.com/bystrogenomics/msgpack-perl.git
+cpanm --quiet --notest MouseX::Getopt
+git clone --depth 1 -branch v0.12 --recurse-submodules https://github.com/salortiz/LMDB_File.git \
+  && cd LMDB_File \
+  && cpanm --quiet .
+  && cd .. \
+  && rm -rf LMDB_File
+cpanm --quiet DBD::mysql@4.051
+```
+
+The remaining dependencies are installed like this:
+
+```bash
+cpanm --installdeps .
```

Comment:
This doesn't work:

```
You have /usr/bin/unzip
--> Working on .
Entering /mnt/ssd1/bystro/perl
Configuring /mnt/ssd1/bystro/perl
-> N/A
! Configuring . failed. See /home/alexkotlar/.cpanm/work/1697325102.82941/build.log for details.
```

I think maybe lack of Makefile?

Reply:
Lots of ways to achieve this, but adding the cpanfile makes it very clear the intention, I think.